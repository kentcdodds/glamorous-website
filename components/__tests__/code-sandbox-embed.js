import React from 'react'
import {mount} from 'enzyme'
import toJson from 'enzyme-to-json'
import {matcher, serializer} from 'jest-glamor-react'
import CodeSandboxEmbed from '../code-sandbox-embed'

expect.addSnapshotSerializer(serializer)
expect.extend(matcher)

test('renders', () => {
  expect(toJson(mount(<CodeSandboxEmbed />))).toMatchSnapshotWithGlamor()
})
