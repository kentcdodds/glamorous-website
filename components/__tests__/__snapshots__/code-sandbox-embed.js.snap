// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<CodeSandboxEmbed>
  <iframe
    sandbox="allow-modals allow-forms allow-popups allow-scripts allow-same-origin"
    src="https://codesandbox.io/embed/undefined"
    style={
      Object {
        "border": 0,
        "borderRadius": 4,
        "height": 500,
        "overflow": "hidden",
        "width": "100%",
      }
    }
  />
</CodeSandboxEmbed>
`;
